prow/config.yaml

```diff
@@ -167,7 +169,7 @@ presubmits:
         - "--service-account=/etc/service-account/service-account.json"
         - "--upload=gs://kubernetes-jenkins/pr-logs"
         - "--" # end bootstrap args, scenario args below
-        - "--build=//cmd/... //pkg/... //federation/... //plugin/... //third_party/... //examples/... //test/... //vendor/k8s.io/..."
+        - "--build=//..."
```

Comment:
master is currently broken, so we shouldn't merge this.

Reply:
I dropped a /hold until https://github.com/kubernetes/kubernetes/pull/53681#issuecomment-336566466 is sorted

Reply:
we need to support negative targets in the scenario; `make bazel-build` is `bazel build -- //... -//vendor/...`.

Reply:
done